jbcli/jbcli/cli/jb.py

```diff
@@ -50,6 +50,9 @@ def cli():
 def add(applications, add_desktop, runtime):
     """Checkout a juicebox app (or list of apps) and load it
     """
+    if not os.path.exists("apps"):
+        print("Please run `jb add` from the root devlandia directory.")
+        return
```

Comment:
instead of putting an error message here, we can just chdir to the correct place.

(this check you added would still pass when the user is in some unrelated directory that happens to have an `apps` directory, like if they're running it from their `fruition` checkout)

```suggestion
    os.chdir(os.path.join(DEVLANDIA_DIR))
```